### DISCUSSION TITLE
AutoRest for Solo Play

### DISCUSSION THREAD

**Discussion by Dalek159** [jhoffmann2/OathUltimateCollection#29] [Q&A] [answered]
Hi, I ran across this in my own modding work and was checking to see if your mod had a solution to it. From what I could tell from reading the code and experimenting with the mod, it does not seem to. If a player is playing solo, then the PlugIn AutoRest doesn't function, as the trigger (onPlayerTurn) is never called by TTS since there is only one player (even if the button is pressed and it makes the turn changed noise). I imagine there must be a way to detect that the player pressed the end turn button, but a quick search hasn't yielded the answer. I know this is a low-priority bug, but in my self-interest, I wanted to log it in the hopes I can leverage your patch.

**Comment by jhoffmann2** [comment, marked as answer]
Hey Dalek yeah there is a solution to this actually 🙂 unfortunately you're right that TTS won't work with the auto rest in solo the way you're used to using it but thankfully there's more than one way to trigger the rest. Either right click your board and select "rest" from the context menu, or literally click the "rest" text on the player board 😎

**Comment by jhoffmann2** [comment]
Alternatively it may actually work better to play in hotseet mode with two players. That way the clockwork prince is registered as a player too and things like the auto round counter will work better
